organizing_hub/mixins.py

```diff
@@ -39,6 +42,28 @@ def handle_no_permission(self):
         )
         return redirect(settings.ORGANIZING_HUB_DASHBOARD_URL)
 
+    def has_feature_access(self):
+        """Return True if we are skiping feature check"""
```

Comment:
Small typo on skipping.

```suggestion
        """Return True if we are skipping feature check"""
```